.github/workflows/pr-commands.yaml

```diff
@@ -53,9 +53,9 @@ jobs:
       - uses: r-lib/actions/pr-push@v1
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-  style-stan:
-    if: startsWith(github.event.comment.body, '/style-stan')
-    name: style-stan
+  stan:
+    if: startsWith(github.event.comment.body, '/stan')
+    name: stan
     runs-on: ubuntu-16.04
```

Comment:
May need to add `sudo apt install clang-format`.

Reply:
good point